#!/bin/sh /usr/share/dpatch/dpatch-run
## rott-1.0-gcc41.patch from Gentoo

@DPATCH@
diff -urNad rott-1.0+dfsg~/rt_actor.h rott-1.0+dfsg/rt_actor.h
--- rott-1.0+dfsg~/rt_actor.h	2008-01-09 13:52:32.000000000 +0100
+++ rott-1.0+dfsg/rt_actor.h	2008-01-10 09:16:01.000000000 +0100
@@ -315,7 +315,6 @@
 extern  void             *actorat[MAPSIZE][MAPSIZE];
 extern  int              angletodir[ANGLES];
 extern _2Dpoint          SNAKEPATH[512];
-extern  int              STOPSPEED;
 extern  int              FRICTION;
 
 extern  int              objcount;
diff -urNad rott-1.0+dfsg~/rt_in.h rott-1.0+dfsg/rt_in.h
--- rott-1.0+dfsg~/rt_in.h	2008-01-09 13:52:32.000000000 +0100
+++ rott-1.0+dfsg/rt_in.h	2008-01-10 09:16:01.000000000 +0100
@@ -147,9 +147,6 @@
 
 extern boolean  Paused;
 extern volatile int LastScan;
-extern KeyboardDef KbdDefs;
-extern JoystickDef JoyDefs[];
-extern ControlType Controls[MAXPLAYERS];
 
 extern boolean  SpaceBallPresent;
 extern boolean  CybermanPresent;
